pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -49,6 +50,19 @@ func NewConnTrackOvsAppCtl(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet
 	if ovsctlClient == nil {
 		return nil
 	}
+
+	cmdOutput, execErr := ovsctlClient.RunAppctlCmd("dpctl/ct-get-maxconns", false)
+	if execErr != nil {
+		klog.Errorf("error when executing dpctl/ct-get-maxconns command: %v", execErr)
+		return nil
+	}
+	maxConns, err := strconv.Atoi(strings.TrimSpace(string(cmdOutput)))
+	if err != nil {
+		klog.Errorf("error when convert dpctl/ct-get-maxconns cmdoutput to int: %v", err)
+		return nil
+	}
+	metrics.MaxConnectionsInConnTrackTable.Set(float64(maxConns))
```

Comment:
here also I assume that this may change dynamically (see comment above for Linux implementation)? maybe we should have an interface method to read the max number of connections (implemented differently based on the platform) and have one background task to update that metric. Or maybe we just refresh the metric every time we read the connections: it may be the simplest solution and my favorite one :) @srikartati

Reply:
Thats a good point @antoninbas. Agree that this number can be changed by admin and we should track this differently. Yes reading max conn metric when reading the connection should be a simple solution. In addition, we should add in all the metrics help statement saying that these metrics are updated every pollInterval. As sometimes depending on poll interval and scrape interval of prometheus server, metrics may not be updated in prometheus server.

Reply:
Thanks, I moved the updating max connection logics inside the DumpFlows() function as a simple solution right now.